ofp4/snvs.p4

```diff
@@ -238,10 +238,12 @@ control SnvsEgress(inout Headers hdr,
 #endif
 
       // Drop loopback.
+#if 0 /* Disabled because comparing multibit values is difficult */
```

Comment:
How do you do it in of?

Reply:
There's a special case in OpenFlow for outputting to the input port: unless you do something special, outputs to the input port are just dropped.

OVN implements this as a very special case for its own logical ports: for each logical port number X, it has a flow that matches the case where the logical input port and the logical output port are both X and drops it.

In a general case, one can match a difference in two 2-bit fields A and B with 2N flows that consists of a pair of flows for each bit, one flow matches A=0, B=1 and the other matching A=1, B=0.

None of these is a great way to do things.

Reply:
If the OF runtime does this automatically, then it's part of the architecture spec and we don't have to do it.
Otherwise we can indeed do as OF: use a switch statement with as many cases as ports. That can be evaluated in only one stage by using an exact table with constant entries. That works if the number is a constant. Otherwise we have to rely on the control-plane populating the table.